### PR TITLE
Resume consumers after reconnect

### PR DESCRIPTION
**Pull request by riiwo**
Only one of the channels was resumed properly

EN-5824

### CODE REVIEW

File: lib/freddy/request.coffee [hunk 1]
```diff
@@ -88,6 +92,10 @@ class Request
 
   _setupResponseQueue: (channel) =>
     @logger.debug "Created response queue for requests"
+    if @isReconnecting
```
Comment (urmastalimaa) [inline lib/freddy/request.coffee:95]: When is this called when `isReconnecting` is false?

Comment (riiwo) [reply lib/freddy/request.coffee:95]: The first time `_setupResponseQueue` is called when `freddy_setup.coffee` calls `prepare` method in this class and then the channel is created. (So when starting up the first time)

**Review by indrekj — APPROVED** [review]
Not entirely sure how well will this work. node-freddy with its amqp lib is not the easiest thing to understand. Especially in regards to rabbitmq reconnections.

AFAIK we use node-freddy for only one message (`kluster.reserve_channel`). It's a RPC call from Engagement-API. It shouldn't be that complicated to use an HTTP request instead. This way we could ignore all the rabbitmq reconnection etc problems in node services and wouldn't need to touch node-freddy at all anymore. I'll let you/team decide if you want to do this though.

**Review by urmastalimaa — APPROVED** [review]
I think this suffices for the problem at hand.

**Pull request merged**